internal/communitymodules/render_test.go

```diff
@@ -0,0 +1,33 @@
+package communitymodules
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestRender(t *testing.T) {
```

Comment:
What if `convertRow...` doesn't work for `moduleMap` with not one row?

Reply:
Added cases for those